homeassistant/components/landisgyr_heat_meter/const.py

```diff
@@ -0,0 +1,181 @@
+"""Constants for the Landis+Gyr Heat Meter integration."""
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.helpers.entity import EntityCategory
+
+DOMAIN = "landisgyr_heat_meter"
+
+HEAT_METER_SENSOR_TYPES = (
+    SensorEntityDescription(
+        key="heat_usage_gj",
+        icon="mdi:fire",
+        name="Heat usage GJ",
+        native_unit_of_measurement="GJ",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+    ),
+    SensorEntityDescription(
+        key="volume_usage_m3",
+        icon="mdi:fire",
+        name="Volume usage m3",
+        native_unit_of_measurement="m3",
```

Comment:
```suggestion
        native_unit_of_measurement=VOLUME_CUBIC_METERS,
```

Please check other sensors too.

Reply:
This wasn't completed for all sensors.